core/src/main/resources/com/pentaho/metaverse/messages/messages.properties

```diff
@@ -73,17 +73,15 @@ INFO.JobOrTrans.Status_1=DRAFT
 INFO.JobOrTrans.Status_2=PRODUCTION
 
 INFO.WrittingGraphForTransformation=Generating graph for transformation {0}
-INFO.TransformationAnalyzeStarting=Analyzer for Transformation {0} - starting
-INFO.TransformationAnalyzeFinished=Analyzer for Transformation {0} - finished
+INFO.TransformationAnalyzeStarting=Data Lineage Analyzer for Transformation {0} - starting
+INFO.TransformationAnalyzeFinished=Data Lineage Analyzer for Transformation {0} - finished
 
 INFO.WrittingGraphForJob=Generating graph for job {0}
-INFO.JobAnalyzeStarting=Analyzer for Job {0} - starting
-INFO.JobAnalyzeFinished=Analyzer for Job {0} - finished
+INFO.JobAnalyzeStarting=Data Lineage Analyzer for Job {0} - starting
+INFO.JobAnalyzeFinished=Data Lineage Analyzer for Job {0} - finished
 
 DEBUG.Locator.StopScan=Locator type {0}: stopScan()
 DEBUG.Locator.StartScan=Locator type {0}: startScan()
 DEBUG.Lineage.noProfileOutputStream=No profile output stream associated with this LineageWriter
 DEBUG.Lineage.noGraphOutputStream=No graph output stream associated with this LineageWriter
 #DEBUG.FallingBackToFirstSetOfInputFields=Step to use to determine which step to use for previous fields is unknown, choosing {0}.
-
-TITLE.Metaverse=Metaverse
```

Comment:
Was the Title.Metaverse removed based on not being used? Just want to double check it was not removed unintentionally

Reply:
It's no longer used. I removed it during this PR.